Documentation/installation/k8s-install-migration.rst

```diff
@@ -193,6 +193,34 @@ Preparation
         $ cilium install --helm-values values-migration.yaml --helm-auto-gen-values values-initial.yaml
         $ cat values-initial.yaml
 
+.. note::
+In case you are migrating from Calico you need to ensure that Calico doesn't use Cilium's interfaces during the migration.
```

Comment:
```suggestion
In case you are migrating from Calico you need to ensure that Calico doesn't accidently switch Cilium's interfaces during the migration.
```